cirq/ops/raw_types.py

```diff
@@ -578,8 +578,15 @@ def _resolve_parameters_(self, resolver):
 
     def _circuit_diagram_info_(self, args: 'cirq.CircuitDiagramInfoArgs'
                               ) -> 'cirq.CircuitDiagramInfo':
-        return protocols.circuit_diagram_info(self.sub_operation, args,
-                                              NotImplemented)
+        sub_op_info = protocols.circuit_diagram_info(self.sub_operation, args,
+                                                     NotImplemented)
+        # Add tag to wire symbol if it exists.
+        if (sub_op_info is not NotImplemented and args and args.include_tags and
```

Comment:
I still don't think you need the check for args.  The type annotation guarantees this is not None.

Reply:
Removed.